src/server/blockchain/AdminWallet.js

```diff
@@ -358,39 +362,16 @@ export class Wallet {
   /**
    * top wallet if needed
    * @param {string} address
-   * @param {moment.Moment} lastTopping
-   * @param {boolean} force
    * @returns {PromiEvent<TransactionReceipt>}
    */
-  async topWallet(
-    address: string,
-    lastTopping?: moment.Moment = moment().subtract(1, 'day'),
-    force: boolean = false
-  ): PromiEvent<TransactionReceipt> {
-    let daysAgo = moment().diff(moment(lastTopping), 'days')
-    if (conf.env !== 'development' && daysAgo < 1) throw new Error('Daily limit reached')
-    try {
-      let userBalance = await this.web3.eth.getBalance(address)
-      let maxTopWei = parseInt(web3Utils.toWei('1000000', 'gwei'))
-      let toTop = maxTopWei - userBalance
-      log.debug('TopWallet:', { userBalance, toTop })
-      if (toTop > 0 && (force || toTop / maxTopWei >= 0.75)) {
-        let res = await this.sendNative({
-          from: this.address,
-          to: address,
-          value: toTop,
-          gas: defaultGas,
-          gasPrice: defaultGasPrice
-        })
-        log.debug('Topwallet result:', { res })
-        return res
+  async topWallet(address: string): PromiEvent<TransactionReceipt> {
+    const tx: TransactionReceipt = await this.sendTransaction(this.proxyContract.methods.topWallet(address)).catch(
```

Comment:
why were all the checks removed?

Reply:
done in https://github.com/GoodDollar/GoodServer/pull/139/commits/113e7728f6d745c5e872c0ddfb275e9e9390919a

Reply:
> why were all the checks removed?

I returned all checks